src/pages/Test/CreateTest.tsx

```diff
@@ -599,9 +602,11 @@ const CreateTest = () => {
             >
               <CustomDateRangePicker
                 showTime={true}
-                onChange={(props: any) => setTestDateRange(props)}
+                onChange={(props: any) => {
+                  console.log("Answer is ",testDateRange);
+                  setTestDateRange(props)}}
                 value={testDateRange}
-                disablePrevDates={true}
+                disablePrevDates={true} // it should be true only
```

Comment:
Booleans could be passed simply without having to explicitly write `true`
```suggestion
               disablePrevDates
```

Reply:
I have not made any changes here, the changes were made only in CustomDateRangePicker.tsx file